Probesys/bileto
# dev: Refactor edit ticket actors with Symfony Form

## Related issue(s)



https://github.com/Probesys/bileto/issues/551

## How to test manually



- Create a ticket
- Edit the actors of the ticket
- Assign yourself the ticket
- Verify everything works correctly

## Checklist
